Packs/OpsGenie/Integrations/OpsGenieV3/OpsGenieV3.py

```diff
@@ -1015,7 +941,8 @@ def fetch_incidents_command(client: Client,
                                                                                      tags,
                                                                                      client.list_incidents,
                                                                                      now,
-                                                                                     demisto.get(last_run, f"{INCIDENT_TYPE}"))
+                                                                                     demisto.get(last_run,
```

Comment:
if not needed, please revert

Reply:
was getting an error from linter. Can this be merged anyway?